src/app/content/challenges/anchor-escrow/en/challenge.mdx

```diff
@@ -38,7 +38,7 @@ cargo add anchor-spl
 ```
 </Codeblock>
 
-Since we're using `anchor-spl`, we also need to update the `Cargo.toml` file to include `anchor-spl/idl-build` in the `idl-build` feature. 
+Since we're using `anchor-spl`, we also need to update the `programs/Cargo.toml` file to include `anchor-spl/idl-build` in the `idl-build` feature. 
```

Comment:
How about taking this one step further and spell out the entire relative path?

```suggestion
Since we're using `anchor-spl`, we also need to update the `programs/blueshift_anchor_escrow/Cargo.toml` file to include `anchor-spl/idl-build` in the `idl-build` feature. 
```

Reply:
great point!